lib/matplotlib/gridspec.py

```diff
@@ -498,3 +498,43 @@ def __eq__(self, other):
 
     def __hash__(self):
         return hash((self._gridspec, self.num1, self.num2))
+
+    def subgridspec(self, nrows, ncols, **kwargs):
+        """
+        Return a `.GridSpecFromSubplotSpec` that has this subplotspec as
+        a parent.
+
+        Parameters
+        ----------
+        nrows : int
+            Number of rows in grid.
+
+        ncols : int
+            Number or columns in grid.
+
+        Returns
+        -------
+        gridspec : `.GridSpec`
+
+        Other Parameters
+        ----------------
+        *kwargs* are passed to `.GridSpec`.
```

Comment:
Must be formatted like parameters:
~~~
    Other Parameters
    ----------------
    **kwargs
        All other parameters are passed to `.GridSpec`.